src/communication/network_interface_cl.c

```diff
@@ -8124,6 +8124,285 @@ logtb_dump_trantable (FILE * outfp)
 #endif /* !CS_MODE */
 }
 
+/*
+ * mmon_get_server_info -
+ *
+ * return: cubrid error
+ *
+ *   server_info(in/out): save memory usage information of the server
+ */
+int
+mmon_get_server_info (MMON_SERVER_INFO & server_info)
+{
+#if defined(CS_MODE)
+  char *buffer, *ptr, *temp_str;
+  int bufsize = 0;
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  int req_error, ival;
+  int error = NO_ERROR;
+
+  req_error =
+    net_client_request2 (NET_SERVER_MMON_GET_SERVER_INFO, NULL, 0, reply,
+			 OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, &buffer, &bufsize);
+  if (req_error)
+    {
+      assert (er_errid () != NO_ERROR);
+      error = er_errid ();
+    }
+  else
+    {
+      ptr = reply;
+      ptr = or_unpack_int (ptr, &ival);
+      ptr = or_unpack_int (ptr, &ival);
+      error = (int) ival;
+    }
+
+  if (error == NO_ERROR)
+    {
+      ptr = or_unpack_string_nocopy (buffer, &temp_str);
+      memcpy (server_info.name, temp_str, strlen (temp_str) + 1);
+      ptr = or_unpack_int64 (ptr, (int64_t *) & (server_info.total_mem_usage));
+    }
+  free_and_init (buffer);
+
+  return error;
+#else /* CS_MODE */
+  er_log_debug (ARG_FILE_LINE, "mmon_get_server_info: THIS IS ONLY a C/S function");
+  return ER_FAILED;
+#endif /* !CS_MODE */
```

Comment:
There seems a way to deal with the case in other utilities. Have you considered this way?
```
#else /* CS_MODE */
  /* Cannot run in standalone mode */
  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_NOT_IN_STANDALONE, 1, "checkpoint");

  return ER_NOT_IN_STANDALONE;
#endif /* !CS_MODE */
```